beancount_reds_importers/libtransactionbuilder/investments.py

```diff
@@ -136,6 +142,8 @@ def get_target_acct(self, transaction, ticker):
             return target
         if transaction.type == 'income' and getattr(transaction, 'income_type', None) == 'DIV':
             return self.target_account_map.get('dividends', None)
+        if transaction.type in ["other", "credit", "debit", "cash", "dep", "transfer"] and 'transfer' not in self.config:
```

Comment:
Is this line necessary? Won't the line below will take care of it?

Reply:
Yep. I'll remove.